docs/2023/puzzles/day17.md

```diff
@@ -2,12 +2,275 @@ import Solver from "../../../../../website/src/components/Solver.js"
 
 # Day 17: Clumsy Crucible
 
+by [@stewSquared](https://github.com/stewSquared)
+
 ## Puzzle description
 
 https://adventofcode.com/2023/day/17
 
+## Solution Summary
+
+This is a classic search problem with an interesting restriction on state transformations.
+
+We will solve this using Djikstra's Algorithm to find a path through the grid, using the heat loss of each position as our node weights. However, the states in our priority queue will need to include more than just position and accumulated heat loss, since the streak of forward movements in a given direction affects which positions are accessible from a given state.
```

Comment:
```suggestion
We will solve this using Dijkstra's Algorithm to find a path through the grid, using the heat loss of each position as our node weights. However, the states in our priority queue will need to include more than just position and accumulated heat loss, since the streak of forward movements in a given direction affects which positions are accessible from a given state.
```